src/send.c

```diff
@@ -174,7 +174,7 @@ iterator_t *send_init(void)
 		    zconf.bandwidth, pkt_len / 8, zconf.rate);
 	}
 	// log rate, if explicitly specified
-	if (zconf.rate <= 0) {
+	if (zconf.rate < 0) {
```

Comment:
Is rate ever below zero, or was there some weird casting going on where values < 0.5 were being rounded down to zero?

Reply:
It looks `zconf.rate` is an integer.

Reply:
Looking at L160 above it looks like this should be fine. @ajholland does this also look alright to you?

Reply:
Lgtm!